contrib/kubernetes/skydive.yaml

```diff
@@ -49,7 +49,7 @@ metadata:
 data:
   SKYDIVE_ANALYZER_FLOW_BACKEND: elasticsearch
   SKYDIVE_ANALYZER_TOPOLOGY_BACKEND: elasticsearch
-  SKYDIVE_ANALYZER_TOPOLOGY_PROBES: k8s
+  SKYDIVE_ANALYZER_TOPOLOGY_PROBES: k8s ovn
```

Comment:
If OVN is not deployed, Skydive will constantly log "failed to connect: dial unix /var/run/ovn/ovnnb_db.sock: connect: no such file or directory". IIRC, we have a mechanism for Docker where we log only once but keep silently retrying in the background. I can have a look at it

Reply:
This behaviour is now fixed